### PR TITLE
feat(sentry): [ABI-4815] Including Sentry configuration and handle errors

### PR DESCRIPTION
**Pull request by elenitaex5**
# Description
Including Sentry configuration for React Native project and handle errors in Logger.

# Checklist:

- [X] Storybook has been updated (if needed) and still working

- [X] Tested in Android simulator
- [X] Tested in iOS simulator
- [ ] Tested in Android device through Expo Go
- [ ] Tested in Android device as app
- [ ] Tested in iOS device through Expo Go
- [ ] Tested in iOS device as app
- [ ] Tested in web (is not working)

### CODE REVIEW

File: src/api/models/user.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+export enum UserGender {
+  MALE = 'male',
+  FEMALE = 'female'
+}
```
Comment (ylagef) [inline src/api/models/user.ts:4]: ¿Other?

File: src/api/user/models.ts [hunk 2]
```diff
@@ -1,12 +1,9 @@
-export interface UserRequest {
-  userId: string
-}
-
 export enum UserGender {
   MALE = 'male',
   FEMALE = 'female'
 }
```
Comment (ylagef) [inline src/api/user/models.ts:4]: ¿Other? (At least on widget they can select other as gender)

File: package.json
Comment (ylagef) [inline package.json:1]: Please refactor devDependencies/dependencies in order to only have the needed in production as dependencies.

**Review by ylagef — CHANGES_REQUESTED** [review]

File: package.json
Comment (elenitaex5) [reply package.json:1]: changed

**Pull request merged**